src/api/utils/puppet.js

```diff
@@ -32,28 +34,32 @@ const puppet = async (url, params) => {
     const page = await browser.newPage();
     const status = await page.goto(url);
     if (!status.ok) {
-      throw new Error('cannot open google.com');
-    }
-    logger('wait');
-    let sec = 5000;
-    let scCnt = scroll ? 6 : 3;
-    logger(scroll);
-    await new Promise(resolve => setTimeout(() => resolve(), sec));
-    for (let i = 0; i < scCnt; i += 1) {
-      await page.evaluate(sc => {
-        window.scrollBy(0, 200);
-        const s = document.getElementById(sc);
-        if (s) {
-          s.scrollTop += 200;
-        }
-      }, scroll);
+      logger('cannot open google.com');
+    } else {
+      logger('wait');
+      let sec = 5000;
+      const scCnt = scroll ? 6 : 3;
+      logger(scroll);
+      await new Promise((resolve) => setTimeout(() => resolve(), sec));
+      for (let i = 0; i < scCnt; i += 1) {
+        // eslint-disable-next-line no-await-in-loop,no-loop-func
+        await page.evaluate((sc) => {
+          // eslint-disable-next-line no-undef
+          window.scrollBy(0, 200);
+          // eslint-disable-next-line no-undef
+          const s = document.getElementById(sc);
```

Comment:
```suggestion
          // global: window
          // global: document
          window.scrollBy(0, 200);
          const s = document.getElementById(sc);
```
Also, you shouldn't use one-letter-vars!